src/ontology/mondo-ingest.Makefile

```diff
@@ -465,3 +475,23 @@ help:
 	@echo "Creates a report of statistics for mapped deprecated terms (docs/reports/mapped_deprecated.md) and pages for each ontology which list their deprecated terms with existing xrefs in Mondo.\n"
 	@echo "mapped-deprecated-terms"
 	@echo "Creates a report of statistics for mapped deprecated terms (docs/reports/mapped_deprecated.md) and pages for each ontology which list their deprecated terms with existing xrefs in Mondo. Also creates a reports/%_mapped_deprecated_terms.robot.template.tsv for all ontologies.\n"
+	@echo "reports/%_term_exclusions.txt"
```

Comment:
FYI: `make` `help`: I added some docs for how to update the OMIM exclusions with non-diseases, but I the docs for running the exclusion tables generally were missing, so I added those.